spec/lib/msf/core/rpc/v10/rpc_console_spec.rb

```diff
@@ -5,6 +5,7 @@
   include_context 'Msf::Simple::Framework'
   include_context 'Metasploit::Framework::Spec::Constants cleaner'
   include_context 'Msf::Framework#threads cleaner', verify_cleanup_required: false
+  include_context 'wait_for_expect'
```

Comment:
I will also add `spec/lib/msf/core/rpc/v10/rpc_session_spec.rb` that is based on this file, this PR is still in draft.